frontend/src/scenes/error-tracking/errorTrackingActivityDescriber.tsx

```diff
@@ -0,0 +1,88 @@
+import {
+    ActivityChange,
+    ActivityLogItem,
+    ChangeMapping,
+    defaultDescriber,
+    Description,
+    HumanizedChange,
+    userNameForLogItem,
+} from 'lib/components/ActivityLog/humanizeActivity'
+import { SentenceList } from 'lib/components/ActivityLog/SentenceList'
+
+import { ErrorTrackingGroup } from '~/queries/schema'
+import { ActivityScope } from '~/types'
+
+const errorTrackingGroupActionsMapping: Record<
+    keyof ErrorTrackingGroup,
+    (change?: ActivityChange, logItem?: ActivityLogItem) => ChangeMapping | null
+> = {
+    assignee: () => null,
+    merged_fingerprints(change: ActivityChange | undefined): ChangeMapping | null {
+        const lenBefore = Array.isArray(change?.before) ? change?.before.length || 0 : 0
+        const lenAfter = Array.isArray(change?.after) ? change?.after.length || 0 : 0
+        const mergeCountChange = lenAfter - lenBefore
+        return {
+            // TODO need to be able to name the group here?
+            description: [<>merged {mergeCountChange} groups into this one.</>],
```

Comment:
is there a good way to get the "name" of a group?

we need to be able to record either at the time of the change or load here so we can say "merged two fingerprints into 'this particular group'" so it makes sense to humans

or have some generic text that's better than "this one" it needs to work in the specific to a group view and the all activity view

Reply:
Right now the "name" is just the `$exception_type` property on a given event which probably isn't the best name even if it was easily accessible. I think we'll be moving towards a world where each group has an associated `$issue_id`. It might be nice to link in that way e.g. `merged {mergeCountChange} groups into #83477`

Reply:
Right now the "name" is just the `$exception_type` property on a given event which probably isn't the best name even if it was easily accessible. I think we'll be moving towards a world where each group has an associated `$issue_id`. It might be nice to link in that way e.g. `merged {mergeCountChange} groups into #83477`